test/forge/HolographDropERC721.t.sol

```diff
@@ -372,34 +365,48 @@ contract HolographDropERC721Test is Test {
     erc721Drop.init(abi.encode(contractName, contractSymbol, contractBps, eventConfig, skipInit, initCode));
   }
 
-  function test_SubscriptionEnabled() public factoryWithSubscriptionAddress(ownedSubscriptionManager) {
-    HolographerInterface holographerInterface = HolographerInterface(address(erc721Drop));
-    HolographDropERC721 customSource = HolographDropERC721(payable(holographerInterface.getSourceContract()));
-
-    IOperatorFilterRegistry operatorFilterRegistry = IOperatorFilterRegistry(
-      0x000000000000AAeB6D7670E522A718067333cd4E
-    );
-    vm.startPrank(address(0x666));
-    operatorFilterRegistry.updateOperator(ownedSubscriptionManager, address(0xcafeea3), true);
-    vm.stopPrank();
-    vm.startPrank(DEFAULT_OWNER_ADDRESS);
-
-    // It should already be registered so turn it off first
-    customSource.manageMarketFilterSubscription(false);
-    // Then turn it on
-    customSource.manageMarketFilterSubscription(true);
-    erc721Drop.adminMint(DEFAULT_OWNER_ADDRESS, 10);
-    HolographERC721 erc721Enforcer = HolographERC721(payable(address(erc721Drop)));
-    erc721Enforcer.setApprovalForAll(address(0xcafeea3), true);
-    vm.stopPrank();
-    vm.prank(address(0xcafeea3));
-    vm.expectRevert(abi.encodeWithSelector(IHolographDropERC721.OperatorNotAllowed.selector, address(0xcafeea3)));
-    erc721Enforcer.transferFrom(DEFAULT_OWNER_ADDRESS, address(0x666), FIRST_TOKEN_ID);
-    vm.prank(DEFAULT_OWNER_ADDRESS);
-    customSource.manageMarketFilterSubscription(false);
-    vm.prank(address(0xcafeea3));
-    erc721Enforcer.transferFrom(DEFAULT_OWNER_ADDRESS, address(0x666), FIRST_TOKEN_ID);
-  }
+  // TODO: Fix this test. It's failing with the error [FAIL. Reason: ERC721: not approved sender] test_SubscriptionEnabled() (gas: 4685864)
```

Comment:
@ACC01ADE this is unrelated to the fee logic, but I noticed these were failing. I was having some trouble debugging. Can you take a look when you have time, and maybe you'll spot something I'm missing 🙏